webpack.common.js

```diff
@@ -36,6 +36,13 @@ module.exports = {
                     },
                 },
             },
+            {
+                test: require.resolve('jquery'),
+                use: [{
+                    loader: 'expose-loader',
+                    options: '$',
```

Comment:
should we also expose `jQuery` as an option ?

Reply:
Something like this 
```
{
    test: require.resolve('jquery'),
    use: [{
      loader: 'expose-loader',
      options: 'jQuery'
    },{
      loader: 'expose-loader',
      options: '$'
    }]
  }
```

Reply:
Sure!

Reply:
Couldn't get it working, will fix in a follow up.